organizations/tests.py

```diff
@@ -5,12 +5,16 @@
 
 
 class OrganizationTestCase(BaseTestCase):
-    def setUp(self):
-        super().setUp()
+    @classmethod
+    def setUpTestData(self):
+        super().setUpTestData()
         # seed some organizations
         Organization.objects.create(name="Org 1", shortcode="org-1")
         Organization.objects.create(name="Org 2", shortcode="org-2")
 
+    def setUp(self):
+        super().setUp()
```

Comment:
this is not needed. If you don't define a function in an inherited class, it would do this by default.